test/powershell/engine/Basic/Assembly.LoadFrom.ps1

```diff
@@ -0,0 +1,66 @@
+Describe "Assembly.LoadFrom Validation Test" -Tags "CI" {
+    BeforeAll {
+        $ConsumerCode = @'
+            using System;
+            using Assembly.Bar;
+
+            namespace Assembly.Foo
+            {
+                public class Consumer
+                {
+                    public static string GetName()
+                    {
+                        return Provider.GetProviderName();
+                    }
+                }
+            }
+'@
+        $ProviderCode = @'
+            using System;
+
+            namespace Assembly.Bar
+            {
+                public class Provider
+                {
+                    public static string GetProviderName()
+                    {
+                        return "Assembly.Bar.Provider";
+                    }
+                }
+            }
+'@
+
+        $TempPath = [System.IO.Path]::GetTempFileName()
```

Comment:
Once the assembly is loaded, the file cannot be removed. If I use `$testdrive`, Pester will try to remove it afterward and result in non-failure errors.